DEV_GUIDE.md

```diff
@@ -345,8 +345,11 @@ the container engine. Default value: `$HOME/.docker/config.json`
 * `VAULT_CHART_VERSION`: version of Vault Helm Chart to be used by the System Tests for Record Encryption. Default value: `0.27.0`
 * `SKIP_STRIMZI_INSTALL`: skip strimzi installation. Default value: `false`
 * `KAFKA_CLIENT`: client used to produce/consume messages. Default value: `strimzi_test_client`. Currently supported values: `strimzi_test_client`, `kaf`, `kcat`
-* `AWS_ACCESS_KEY_ID`: key id of the aws account to be used for KMS. Localstack will be used by default. Default value: `test`
-* `AWS_SECRET_ACCESS_KEY`: secret access key of the aws account to be used for KMS. Localstack will be used by default. Default value: `test` 
+* `AWS_USE_CLOUD`: set to `true` in case AWS Cloud is used for Record Encryption System Tests. LocalStack will be used by default. Default value: `false`
+* `AWS_ACCESS_KEY_ID`: key id of the aws account with admin permissions to be used for KMS management. Mandatory when `AWS_USE_CLOUD` is `true`. Default value: `test`
```

Comment:
Naming not too clear.

How making it clear the scoping to record encryption.  Also suggest MANAGEMENT for the key used for key management and KMS for the key used by the KMS.

RECORD_ENCRYPTION_AWS_USE_CLOUD
RECORD_ENCRYPTION_MANAGEMENT_AWS_SECRET_ACCESS_KEY
RECORD_ENCRYPTION_KMS_AWS_SECRET_ACCESS_KEY

Reply:
I think its valid concern but I'm worried the PR discussions are turning into bikeshedding.

Reply:
OK